### DISCUSSION TITLE
How to implement control swarm techniques developed on Matlab for the crazy flies?

### DISCUSSION THREAD

**Discussion by frulloa** [bitcraze/discussions#621] [Q&A]
What is the better way to implement control swarm techniques developed on Matlab for the crazy flies?
The laboratory have optitrack to detect the crazyflies movement.

**Comment by krichardsson** [comment]
The easiest way is probably to use [Crazyswarm](https://crazyswarm.readthedocs.io/en/latest/), it supports mocap positioning and ROS. You will probably have to re-implement your matlab code in python for instance. It might be possible to use matlab in some way, but I do not know anything about this.
